### PR TITLE
refactor: merge readme files

### PR DESCRIPTION
**Pull request by slettner**
Closes https://github.com/jina-ai/examples/issues/576

### CODE REVIEW

File: wikipedia-sentences/README.md [hunk 1]
```diff
@@ -62,12 +62,47 @@ To quickly get started, you can index a [small dataset of 50 sentences](data/toy
...
+For this case, Jina provides a simple and intuitive solution which we will demonstrate using a second [small dataset](data/toy-input-incremental.txt).
+Just as before, index the [first dataset](data/toy-input.txt) and then *incrementally* the [second dataset](data/toy-input-incremental.txt)
+```python
+with Flow().load_config('flows/index.yml'):
+    f.index_lines(filepath='data/toy-input.txt', read_mode='r', batch_size=16, num_docs=10)
+    f.index_lines(filepath='data/toy-input-incremental.txt', read_mode='r', batch_size=16, num_docs=10)
+```
+One challenge we need to address when incrementally adding to the index is duplication of Documents.
+Jina provides a [DocCache](pods/index_cache.yml) pod that is pre-configured for you and takes care of detecting duplicates 
+when adding to the index. Add one line to your [index flow](flows/index_incremental.yml) and incremental indexing works.
```
Comment (alexcg1) [inline wikipedia-sentences/README.md:90]: - flow -> Flow

File: wikipedia-sentences/README.md [hunk 2]
```diff
@@ -62,12 +62,47 @@ To quickly get started, you can index a [small dataset of 50 sentences](data/toy
...
+Of course, we don't want to re-calculate our index for all our data every time we add a couple of new Documents. 
+For this case, Jina provides a simple and intuitive solution which we will demonstrate using a second [small dataset](data/toy-input-incremental.txt).
+Just as before, index the [first dataset](data/toy-input.txt) and then *incrementally* the [second dataset](data/toy-input-incremental.txt)
+```python
+with Flow().load_config('flows/index.yml'):
+    f.index_lines(filepath='data/toy-input.txt', read_mode='r', batch_size=16, num_docs=10)
+    f.index_lines(filepath='data/toy-input-incremental.txt', read_mode='r', batch_size=16, num_docs=10)
+```
+One challenge we need to address when incrementally adding to the index is duplication of Documents.
+Jina provides a [DocCache](pods/index_cache.yml) pod that is pre-configured for you and takes care of detecting duplicates
```
Comment (alexcg1) [inline wikipedia-sentences/README.md:89]: ## Capitalization

Please capitalize first letters of the following (when used to reference project or component):

- Flow
- Pod
- Pea
- Executor
- Driver
- Jina
- Jina Hub
- Jina Dashboard
- Jina Box

### Note!

This only applies **outside** a code or CLI context. If you're describing Python, YAML, any programming language, or command line input/output, please use the capitalization you would use **in that context**.

##### Right

```python
from jina.flow import Flow
```

##### Wrong

```python
from Jina.Flow import Flow
```

---

Made with [GitHub saved replies](https://github.com/settings/replies)

File: wikipedia-sentences/README.md [hunk 3]
```diff
@@ -62,12 +62,47 @@ To quickly get started, you can index a [small dataset of 50 sentences](data/toy
 ```sh
 python app.py -t index
 ```
-If you see the following output, it means your data has been correctly indexed. 
+The relevant Jina code to index data given your flow's YAML definition breaks down to
+```python
+with Flow().load_config('flows/index.yml'):
+    f.index_lines(filepath='data/toy-input.txt', read_mode='r', batch_size=16, num_docs=10)
+```
+The flow will interpret each line in the txt file as one Document.
```
Comment (alexcg1) [inline wikipedia-sentences/README.md:70]: Flow

File: wikipedia-sentences/README.md (hunk 2 above)
Comment (alexcg1) [reply wikipedia-sentences/README.md:89]: Several places have capitalization issues in this doc. Some I've noticed, some I may not have. So just give it a once over and check it fits the caps guide

**Review by alexcg1 — CHANGES_REQUESTED** [review]

File: wikipedia-sentences/README.md (hunk 1 above)
Comment (slettner) [reply wikipedia-sentences/README.md]: Fixed in https://github.com/jina-ai/examples/pull/577/commits/fd98b7492d5df999878f1f3526b9fce2ba3f8943

File: wikipedia-sentences/README.md (hunk 2 above)
Comment (slettner) [reply wikipedia-sentences/README.md]: Thanks for the guide Alex, I scanned the doc and hope to resolved all capitalization issues.

File: wikipedia-sentences/README.md (hunk 3 above)
Comment (slettner) [reply wikipedia-sentences/README.md]: Fixed in https://github.com/jina-ai/examples/pull/577/commits/fd98b7492d5df999878f1f3526b9fce2ba3f8943

File: wikipedia-sentences/README.md [hunk 4]
```diff
@@ -62,12 +62,47 @@ To quickly get started, you can index a [small dataset of 50 sentences](data/toy
...
+The Flow will interpret each line in the txt file as one Document.
+You can limit the number of indexed Documents with the `num_docs`
+argument.
+If you see the following output, it means your data has been correctly indexed.
 ```sh
 Flow@5162[S]:flow is closed and all resources are released, current build level is 0
 ```
 
-We recommend you come back to this step later and index the full wikipedia dataset for better results. To index the [full dataset](https://www.kaggle.com/mikeortman/wikipedia-sentences) (almost 900 MB) follow these steps:
+#### What if new data arrives that needs to be indexed?
```
Comment (FionnD) [inline wikipedia-sentences/README.md:78]: Could we add this section as **[Optional] Step 5.** 

I see the logic of adding it in the indexing section, but my primary concern is getting the example running with minimum interference. Once it's running, and they get that 'woohoo' moment, we can build on top of it.

File: wikipedia-sentences/README.md [hunk 5]
```diff
@@ -62,12 +62,47 @@ To quickly get started, you can index a [small dataset of 50 sentences](data/toy
...
+Many applications will require incremental indexing, i.e. a way to re-use an index to add new data.
```
Comment (FionnD) [inline wikipedia-sentences/README.md:79]: **suggestion**: replace 
_i.e. a way to re-use an index to add new data_ 
with 
_which is a way to add new data to an index, without re-indexing the original data._ 

Reason: Just sounds better in my head.

File: wikipedia-sentences/README.md [hunk 6]
```diff
@@ -62,12 +62,47 @@ To quickly get started, you can index a [small dataset of 50 sentences](data/toy
...
+For this case, Jina provides a simple and intuitive solution which we will demonstrate using a second [small dataset](data/toy-input-incremental.txt).
+Just as before, index the [first dataset](data/toy-input.txt) and then *incrementally* the [second dataset](data/toy-input-incremental.txt)
+```python
+with Flow().load_config('flows/index.yml'):
+    f.index_lines(filepath='data/toy-input.txt', read_mode='r', batch_size=16, num_docs=10)
+    f.index_lines(filepath='data/toy-input-incremental.txt', read_mode='r', batch_size=16, num_docs=10)
+```
+One challenge we need to address when incrementally adding to the index is duplication of Documents.
+Jina provides a [DocCache](pods/index_cache.yml) Pod that is pre-configured for you and takes care of detecting duplicates 
+when adding to the index. Add one line to your [index Flow](flows/index_incremental.yml) and incremental indexing works.
```
Comment (FionnD) [inline wikipedia-sentences/README.md:90]: when adding new data to the index.

File: wikipedia-sentences/README.md [hunk 7]
```diff
@@ -62,12 +62,47 @@ To quickly get started, you can index a [small dataset of 50 sentences](data/toy
...
+pods:
+  - name: encoder
+    uses: pods/encode.yml
+    timeout_ready: 1200000
+    read_only: true
+  - name: indexer
+    uses_before: pods/index_cache.yml  # use before indexing to detect duplicates 
+    uses: pods/index.yml
+```
+
```
Comment (FionnD) [inline wikipedia-sentences/README.md:103]: **Issue**: We don't provide instructions to explain how to run the incremental function. It should be 

`python app.py -t index_incremental `

right?

File: wikipedia-sentences/README.md (hunk 6 above)
Comment (FionnD) [reply wikipedia-sentences/README.md:90]: **Suggestion**: 
_Add one line to your [index Flow](flows/index_incremental.yml) and incremental indexing works._

To me, this sentence implies the user must do this action, but it's already done for them. Could we clarify this with something like. As you can see in this file... all we did was change x

File: wikipedia-sentences/README.md [hunk 8]
```diff
@@ -154,13 +189,13 @@ Here is a small overview if you're interested in understanding what each file in
 |File   | Explanation  |
 |--|--|
 |📂 `flows/`  | Folder to store Flow configuration    |
-|--- 📃 `flows/index.yml`  | Contains the details of which executors should be used for indexing your data. |
-|--- 📃 `flows/query.yml`  | Contains the details of which executors should be used for querying your data. |
+|--- 📃 `flows/index.yml`  | Contains the details of which Executors should be used for indexing your data. |
```
Comment (FionnD) [inline wikipedia-sentences/README.md:192]: can you add the incremental index flow as a row here please?

**Review by FionnD — CHANGES_REQUESTED** [review]
Great work! 

Don't forget to remove line 215 in the next steps, if you delete the old readme

File: wikipedia-sentences/README.md (hunk 4 above)
Comment (slettner) [reply wikipedia-sentences/README.md]: Yep, that is a good idea. I moved it down (https://github.com/jina-ai/examples/pull/577/commits/ea5f5db0d2e324bcc751aa8cb62fd4b44e9bad62)

File: wikipedia-sentences/README.md (hunk 5 above)
Comment (slettner) [reply wikipedia-sentences/README.md]: Agree (https://github.com/jina-ai/examples/pull/577/commits/ea5f5db0d2e324bcc751aa8cb62fd4b44e9bad62)

File: wikipedia-sentences/README.md (hunk 6 above)
Comment (slettner) [reply wikipedia-sentences/README.md]: Agree, thx https://github.com/jina-ai/examples/pull/577/commits/ea5f5db0d2e324bcc751aa8cb62fd4b44e9bad62

File: wikipedia-sentences/README.md (hunk 7 above)
Comment (slettner) [reply wikipedia-sentences/README.md]: Uh, thanks for noticing! https://github.com/jina-ai/examples/pull/577/commits/ea5f5db0d2e324bcc751aa8cb62fd4b44e9bad62

File: wikipedia-sentences/README.md (hunk 8 above)
Comment (slettner) [reply wikipedia-sentences/README.md:192]: Added in https://github.com/jina-ai/examples/pull/577/commits/ea5f5db0d2e324bcc751aa8cb62fd4b44e9bad62

**Comment by slettner** [comment]
@FionnD Thank you for the Feedback, I tried to work everything in. Would be great if you can have a second look :)

**Review by FionnD — APPROVED** [review]
Looks super to me! Thanks for the quick changes :)

**Review by alexcg1 — APPROVED** [review]
LGTM :+1:

**Pull request merged**